### PR TITLE
Add more json doc information

### PR DESCRIPTION
**Pull request by MariusKl**
add @apierrors to document possible http error codes
refactor GeoCodeCaller and add more tests

### CODE REVIEW

File: src/main/java/de/helfenkannjeder/come2help/server/rest/AbilitiesController.java [hunk 1]
```diff
@@ -57,7 +63,7 @@ public AbilityDto updateAbility(@NotNull @PathVariable Long id, @Valid @RequestB
 
     @RequestMapping(value = "/{id}", method = RequestMethod.DELETE)
     @ResponseStatus(HttpStatus.NO_CONTENT)
-    public void deleteAbility(@NotNull @PathVariable Long id) {
+    public void deleteAbility(@NotNull @PathVariable(value = "id") Long id) {
```
Comment (andreas-eberle) [inline src/main/java/de/helfenkannjeder/come2help/server/rest/AbilitiesController.java]: Why did you add this? It works fine without the extra boilerplate...

Comment (MariusKl) [inline src/main/java/de/helfenkannjeder/come2help/server/rest/AbilitiesController.java]: It is just for jsondoc. Without the api documentation is not completed and the "playground" does not show all parameters properly.

Comment (andreas-eberle) [inline src/main/java/de/helfenkannjeder/come2help/server/rest/AbilitiesController.java]: ok, thanks for the explanation.
